3rdparty/libmgf/src/Parser.cpp

```diff
@@ -828,222 +828,288 @@ namespace mgf {
   case 61:
 /* Line 664 of lalr1.cc  */
 #line 222 "Parser.ypp"
-    { driver.context.currentSpectrum.setIT_MODS(*(yysemantic_stack_[(4) - (3)].stringVal)); delete (yysemantic_stack_[(4) - (3)].stringVal); }
+    { driver.context.currentSpectrum.setINSTRUMENT(*(yysemantic_stack_[(3) - (2)].stringVal)); delete (yysemantic_stack_[(3) - (2)].stringVal); }
```

Comment:
@saifulbkhan Could you explain what these numbers (3), (2) represent?

Reply:
These `.cpp` files are auto-generated. You can instead look at this [highlighted diff](https://github.com/ElucidataInc/ElMaven/pull/1173/files/c1e1030b6825d2a372fb1061f36463e47dc87ca8#diff-64411394478601012964536bd244bd06R221-R222) and see that the numbers represent the tokens captured in order (kind of like regex groups).

Reply:
Got it.